tools/dragen-germline/4.0.3/dragen-germline__4.0.3.cwl

```diff
@@ -72,9 +72,9 @@ requirements:
             --extract \\
             --file "$(inputs.reference_tar.path)"
           
-          # Confirm not both fastq_list and fastq_list_rows are defined
-          if [[ "$(is_not_null(inputs.fastq_list))" == "true" && "$(is_not_null(inputs.fastq_list_rows))" == "true" ]]; then
-            echo "Cannot set both CWL inputs fastq_list AND fastq_list_rows" 1>&2
+          # Confirm either of fastq_list, fastq_list_rows or bam_input is defined
```

Comment:
This should be == "1" over '-gt 1' since we want one and only of fastq_list, fastq_list_rows and bam_input for germline.

Reply:
updated in https://github.com/umccr/cwl-ica/commit/5807359ef8a1d6cad655cbcb7488f949ee3102ef